addons/link_tracker/models/mail_render_mixin.py

```diff
@@ -39,6 +39,7 @@ def _shorten_links(self, html, link_tracker_vals, blacklist=None, base_url=None)
         """
         if not html or is_html_empty(html):
             return html
+        blacklist = self._shorten_links_get_default_blacklist(blacklist)
```

Comment:
Seems you always want to blacklist /cards, so maybe it could be simpler to hardcode it here and in links_text and remove the method and its overrides. Sometimes the simpler the better. Someday unsubscribe_from_list and view will probably be hardcoded also, because anyway we don't have a real need of such modular approach I think.